symfony/translation/3.3/config/packages/translation.yaml

```diff
@@ -1,7 +1,8 @@
 framework:
     default_locale: '%locale%'
     translator:
-        paths:
-            - '%kernel.project_dir%/translations'
+        # additional translation paths
+        #paths: []
+        default_path: '%kernel.project_dir%/translations'
```

Comment:
Are you sure `default_path` works here ? IIRC, it is a Symfony 3.4+ feature, while this recipe applies to 3.3+

Reply:
Yes, but I'm following the same approach that @nicolas-grekas in the previous PR:
> https://github.com/symfony/recipes/pull/416#issuecomment-400427622 (3.3 is EOLed so we don't really care for it)

Reply:
a recipe is for new projects 99%, and no new project will use this recipe ever (or at least we don't care IMHO)

Reply:
but then, it would be better to migrate this recipe to be 3.4+ and have no recipe at all for 3.3, than to have a 3.3+ recipe triggering an error on 3.3